src/tests/suite/dfs_unit_test.c

```diff
@@ -393,6 +393,56 @@ dfs_test_lookup(void **state)
 	assert_int_equal(rc, 0);
 }
 
+static void
+dfs_test_open_stat(void **state)
```

Comment:
instead of adding a new test, please just extend the dfs_test_chown to test also with open_stat.

Reply:
should probably also add an NLT test for dfuse level test

Reply:
Will do.

I've tested dfuse with this code (it's a port of some of the multi-user code) but as yet you can't call it from dfuse yet as it rejects all I/O if the user/group doesn't match entirely.  This PR has the effect of changing how dfuse sets the uid/gid so it's set from the kernel request not get*id in the dfs code, but it is not enough to allow dfuse to set different values.

Reply:
Done.